analysis/src/SignatureHelp.ml

```diff
@@ -113,7 +113,12 @@ let extractParameters ~signature ~typeStrForParser ~labelPrefixLen =
       | {
        (* Gotcha: functions with multiple arugments are modelled as a series of single argument functions. *)
        Parsetree.ptyp_desc =
-         Ptyp_arrow (argumentLabel, argumentTypeExpr, nextFunctionExpr, _);
+         Ptyp_arrow
+           {
+             lbl = argumentLabel;
+             typ1 = argumentTypeExpr;
```

Comment:
Should we maybe use more descriptive names than `typ1` and `typ2`?
Something like `argument_type` and `return_type` (or `arg_typ` and `ret_typ` or similar)?

Reply:
That'd be a good addition indeed.

Reply:
The fact that they are types is redundant, so `typ1` and `typ2` convey very little information.
`arg` and `ret` could work well.

Reply:
Renamed.